lang/po/new/cs.po

```diff
@@ -1193,7 +1190,7 @@ msgstr "Vyber jazyka"
 # MSG_SELFTEST_OK c=20
 #: ultralcd.cpp:7679
 msgid "Self test OK"
-msgstr ""
+msgstr "Self test OK"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.